ldif.go

```diff
@@ -271,6 +277,11 @@ func (l *LDIF) parseEntry(lines []string) (entry *Entry, err error) {
 				if attr != attribute {
 					return nil, fmt.Errorf("invalid attribute %s in %s request for %s", attr, op, attribute)
 				}
+				if values == nil {
```

Comment:
no need for the condition, you can just append to the nil valued []string and the right thing[tm] will happen